localization/locales/en.json

```diff
@@ -12,7 +12,13 @@
       "age_limit": "Info: Stale data (min.)",
       "show_age": "Show data age",
       "calc_trend": "Calc trend",
-      "units_in_mmol": "Units in mmol/l"
+      "units_in_mmol": "Units in mmol/l",
+      "always_on_top": "Always on top"
+    },
+    "shortcut": {
+      "title": "Shortcuts",
+      "bring_to_front": "Bring To Front",
```

Comment:
Why do you need an additional flag for this? You've already managed the visibility state by the `always_on_top` flag. If it is false, you need a way to obtain the app in focus. The only option in your PR is the hot-key combination.